pom.xml

```diff
@@ -451,5 +454,22 @@ Note: The maven build currently needs to be started from the javaslang root dir
                 </pluginManagement>
             </build>
         </profile>
+        <!-- A profile for running the benchmarks -->
+        <profile>
+            <id>benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>${maven.exec.version}</version>
+                        <configuration>
+                            <classpathScope>test</classpathScope>
+                            <mainClass>javaslang.JmhRunner</mainClass>
```

Comment:
Pushed a new version, with `exec:exec` plugin instead of `exec:java` (as we need a forked JVM), with updated documentation and fixed `main`.
It's working currently with
```
mvn clean test -P benchmark -pl javaslang-benchmark
```